src/app/screens/ledger/addStxAddress/index.tsx

```diff
@@ -130,11 +136,10 @@ function AddStxAddress(): JSX.Element {
 
       const stacksCreds = await importStxAccounts(true);
 
-      if (!stacksCreds) {
-        throw new Error('No response');
+      if (stacksCreds) {
+        await saveAddressToWallet(stacksCreds);
       }
-      await saveAddressToWallet(stacksCreds);
-    } catch (err) {
+    } catch (err: any) {
```

Comment:
this behaviour has changed. is this intentional? I would expect that the previous throw new Error, would still want to trigger the state changes in the catch block

Reply:
Yeah, it looks like a mistake while merging git conflicts